sdk/java/pulumi/src/main/java/com/pulumi/deployment/InvokeOutputOptions.java

```diff
@@ -0,0 +1,67 @@
+package com.pulumi.deployment;
+
+import com.pulumi.core.TypeShape;
+import com.pulumi.core.internal.Internal;
+import com.pulumi.core.internal.annotations.InternalUse;
+import com.pulumi.resources.InvokeArgs;
+import com.pulumi.resources.ProviderResource;
+import com.pulumi.resources.Resource;
+
+import javax.annotation.Nullable;
+import javax.annotation.ParametersAreNonnullByDefault;
+import java.util.Optional;
+import java.util.List;
+
+import static java.util.Objects.requireNonNull;
+
+/**
+ * Options to help control the behavior of @see
```

Comment:
I don't think you need/should have `@see` here -- `{@link ...}` should be sufficient